Formula/alerter.rb

```diff
@@ -0,0 +1,23 @@
+class Alerter < Formula
+  desc "Send User Alert Notification on Mac OS X from the command-line"
```

Comment:
```suggestion
  desc "Send User Alert Notification on macOS from the command-line"
```

Reply:
Nothing against this, i just copied alerter's own description